docker/Dockerfile.ocl

```diff
@@ -0,0 +1,64 @@
+# USAGE: sudo docker build libs/tvm -f libs/tvm/docker/Dockerfile.ocl -t l4b/tvm:ocl
+
+# REFERENCE: https://docs.docker.com/engine/reference/builder
+
+FROM ubuntu:18.04
+
+RUN echo "Labelling this image"
+LABEL Description="Docker image for TVM built with OpenCL & OpenGL support"
+LABEL Vendor="L4B Software"
```

Comment:
If we are going to merge it upstream, we need to remove the Vendor and Maintainer flag, as this file is going to be maintained by the community